spectacles/cli.py

```diff
@@ -273,6 +294,13 @@ def _build_base_subparser() -> argparse.ArgumentParser:
         help="Display debug logging during spectacles execution. \
             Useful for debugging and making bug reports.",
     )
+    base_subparser.add_argument(
+        "--log-directory",
+        action=EnvVarAction,
+        env_var="LOG_DIRECTORY",
```

Comment:
Can we do `SPECTACLES_LOG_DIR` to avoid any conflict with non-Spectacles environment variables? Best practice to be app-specific with these.